### DISCUSSION TITLE
Any way to run the .ps1 from CMD/Powershell?

### DISCUSSION THREAD

**Discussion by eudy97** [LesFerch/WinSetView#84] [Q&A] [answered]
if I have a copy of my desired setting in a .ini file, how do I pass it to the .ps1 script using powershell? 

I'm trying to automate the process so no user input is needed. Thanks in advance.

**Comment by LesFerch** [comment, marked as answer]
Did you see the instructions [here](https://lesferch.github.io/WinSetView/Manual.html#command-line-operation)?

**Reply by eudy97** [reply]
100% missed that.

Thank you very much.
